### DISCUSSION TITLE
怎么注册 内置函数 max_by

### DISCUSSION THREAD

**Discussion by JavaCPlusMario** [apache/iotdb#12954] [General]
select max_by(maxTempId, maxTemp) from root.sg.sensor_1; 

Msg: org.apache.iotdb.jdbc.IoTDBSQLException: 701: Error occurred during inferring UDF data type:
java.lang.RuntimeException: Failed to reflect UDF instance, because UDF MAX_BY has not been registered.

咋提升 没有 注册; 
应该 怎么 注册 捏

**Comment by JavaCPlusMario** [comment]
没有找到 注册的办法啊

**Comment by HTHou** [comment]
https://github.com/apache/iotdb/issues/12943 
1.3.2 新加的这个函数，应该不需要注册
